Python.gitignore

```diff
@@ -82,6 +82,11 @@ ipython_config.py
 # pyenv
 .python-version
 
+# pipenv
+#   According to pypa/pipenv#598, it is recommended to include Pipfile.lock in VCS.
+#   However, in case of collaboration, Pipfile.lock may conflict on different OS.
```

Comment:
This comment from the linked issue is helpful context for problems with the lockfile - @JarryShaw are you able to elaborate in this comment on the problems that users might see with this lockfile? https://github.com/pypa/pipenv/issues/598#issuecomment-361811946

Reply:
Just rephrased a bit. Hopefully it is now better :)